README.md

```diff
@@ -464,9 +335,9 @@ terraform-docs markdown table . --output-file README.md
 
 | Name | Version |
 |------|---------|
-| <a name="provider_castai"></a> [castai](#provider\_castai) | 7.3.0 |
-| <a name="provider_helm"></a> [helm](#provider\_helm) | 2.13.2 |
-| <a name="provider_null"></a> [null](#provider\_null) | 3.2.2 |
+| <a name="provider_castai"></a> [castai](#provider\_castai) | ~> 7.3.0 |
+| <a name="provider_helm"></a> [helm](#provider\_helm) | >= 2.0.0 |
+| <a name="provider_null"></a> [null](#provider\_null) | n/a |
```

Comment:
question: these version values seem to differ from the changes introduced in #93 

I generated these automatically with `make generate-doc`. Is there a reason #93 produced different values and that tthe `make generate-doc` produces different values after this revert??

cc @aldor007

Reply:
I would appreciate your take too @linkas45 :pray:

Reply:
I think it was not generated, but changed manually. I would say it's good to go as this release is still blocking customers

Reply:
Okay then, I’ll proceed.